src/components/menu/js/menuDirective.js

```diff
@@ -275,7 +276,7 @@ function MenuDirective($mdUtil) {
 
     element.append(menuContainer);
     menuContainer[0].style.display = 'none';
-    mdMenuCtrl.init(menuContainer, { isInMenuBar: isInMenuBar });
+    mdMenuCtrl.init(menuContainer, { isInMenuBar: isInMenuBar, mdMenuBarCtrl : mdMenuBarCtrl });
```

Comment:
```suggestion
    mdMenuCtrl.init(menuContainer, { isInMenuBar: isInMenuBar, mdMenuBarCtrl: mdMenuBarCtrl });
```

Reply:
Thanks for the suggestion. I'll make sure to apply this in my next change.